sdk/src/main/java/com/ifountain/opsgenie/client/model/BaseResponseWithId.java

```diff
@@ -1,13 +1,15 @@
 package com.ifountain.opsgenie.client.model;
 
+import com.ifountain.opsgenie.client.model.beans.DataWithId;
+
 public class BaseResponseWithId extends BaseResponse {
-    private String id;
+    private DataWithId data;
```

Comment:
DataWIthId only contains an id and their getter setter. are we expecting data in it as well or do we just want to get the id in it? in that case id there should also be fine.

Reply:
We have to set id in this way because of API contracts. Most of the response from APIs are structured in this way only.
{
    "result": "Created",
    "data": {
        "id": "4fb6ad93-ffed-4c04-b62a-f3bfd993fb3d"
    },
    "took": 1.112,
    "requestId": "b5862ad7-d805-4901-8b6d-2ea75dba0f15"
}